Disk indicator troubleshooting (#90504)
elastic/elasticsearch: docs/reference/troubleshooting/disk/fix-master-node-out-of-disk.asciidoc

[[fix-master-node-out-of-disk]]
== Fix master nodes out of disk

> Elasticsearch is using master nodes to coordinate the cluster. If one or more of these nodes are running

Reviewer: ```suggestion
{es} is using master nodes to coordinate the cluster. If the master or any master eligible nodes are running
```

> out of space, you need to ensure that they have enough disk space to function. If the health disk indicator

Reviewer: I think we should link to the health disk indicator docs here

Author: Added the health api reference doc.
